### PR TITLE
Added desktop version

### PR DESCRIPTION
**Pull request by bekjon-me**
- added media queries in CSS
- completed the whole desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -87,7 +100,7 @@ <h2>Profesional Art Printing Data</h2>
               <li>ruby</li>
             </ul>
           </div>
-          <button type="button">See Project</button>
+          <button id="display" type="button">See Project</button>
```
Comment (roshan-bajgain) [inline index.html:103]: - [ ] According to the project requirement you should stick to the design as much as possible. The buttons should be aligned on the left. please implement it
### Your design
your
### Figma design
figma

File: style.css [hunk 2]
```diff
@@ -436,3 +449,199 @@ footer > .social > li > a {
...
+  }
+
+  #display {
+    display: block !important;
+    transform: translateY(-100%);
+  }
+
+  #portfolio > .cards > .card > .main > .skills {
+    margin-top: 24px;
+  }
```
Comment (roshan-bajgain) [inline style.css:590]: - [ ] I would suggest you please stick as much as possible to the design. please implement the same color and same background to stick to the design
- [ ] please implement in all the buttons of this section
### Your design
figma
### Figma design
yours

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @bekjon-me, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (Abenezer-Tilahun) [reply style.css:604]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @bekjon-me      , 

Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional  README
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**